### PR TITLE
feat : added questions page

### PR DESCRIPTION
**Pull request by Priya-Patel31**
It consists of :
* Header
* Main-container
 • All questions

### CODE REVIEW

File: questions_page/questions_page.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  </head>
+  <body>
+    <div class="question-page-container">
+        <div class="header flex-row justify-between align-center px-2">
+            <div class="flex-row justify-center align-center">
+                <img class ="logo-image" src="/images/logo3.png" alt="logo"/>
+                <span class="logo-text font-semibold">Quiz<span class="text-orange">Buzz</span></span>
+            </div>
+            <div class="search-bar-container">
+              <button type="submit">
```
Comment (rkapoor10) [inline questions_page/questions_page.html]: you may remove type="submit" if not in use

File: questions_page/questions_page.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                <input type="radio" name="options" value="mount-kenya" class="my-2 radio-btn"/>&nbsp;Mount Kenya
+              </li>
+              <li>
+                <input type="radio" name="options" value="stanley" class="my-2 radio-btn"/>&nbsp;Stanley
+              </li>
+              <li>
+                <input type="radio" name="options" value="toubkal" class="my-2 radio-btn" />&nbsp;Toubkal
+              </li>
+              <li>
+                <input type="radio" name="options" value="simien" class="my-2 radio-btn" />&nbsp;Simien
```
Comment (sks-21) [inline questions_page/questions_page.html:102]: On hovering, the cursor should be pointer

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge!

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge. Just go through the suggestions.

File: questions_page/questions_page.html (hunk 1 above)
Comment (Priya-Patel31) [reply questions_page/questions_page.html]: made changes in it!!

File: questions_page/questions_page.html (hunk 2 above)
Comment (Priya-Patel31) [reply questions_page/questions_page.html:102]: added it !!

**Pull request merged**